client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -184,14 +184,14 @@ export class DomainWarnings extends React.PureComponent {
 				learnMoreUrl = MAP_SUBDOMAIN;
 			} else {
 				text = translate(
-					"{{strong}}%(domainName)s's{{/strong}} name server records need to be configured.",
+					"ACTION REQUIRED: Please contact your domain registrar to point {{strong}}%(domainName)s's{{/strong}} name server records to WordPress.com.",
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

Reply:
@nerdysandy It looks like you want to use a comment here rather than context, but I"m not 100% on that. I'd ping in #i18n for clarification. There's also a great FG document on the subject, search "Context vs. Comment"